src/elements/TACSElement.h

```diff
@@ -183,6 +183,15 @@ class TACSElement : public TACSObject {
     return NULL;
   }
 
+  /**
+    Create element traction class
+
+    @return The TACSElement traction class associated with this element. Possibly NULL.
+  */
+  virtual TACSElement* createElementTraction(TacsScalar t[], int faceIndex){
```

Comment:
I think the faceIndex argument should come first to be consistent with the way we've typically written these function prototypes.

Reply:
Should be fixed now